### DISCUSSION TITLE
Is there a way to create own categories?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#468] [Q&A]
So for example,if I wanna create the category "Final Fantasy" and only want that all Final Fantasy games are listed there and there is an icon for that [here](https://i.imgur.com/Zt215W9.png),is that possible?

**Comment by neuromancer** [comment]
There is no way to create new "categories", but you can always create tags to group games.

**Reply by ghost** [reply]
Can I search for games that have the same tag?

**Reply by tkashkin** [reply]
@Toadfield click the Filters button in the headerbar and select tags you want to show.

**Reply by ghost** [reply]
thanks
